src/app_service/service/chat/dto/chat.nim

```diff
@@ -416,3 +423,12 @@ proc communityChannelUuid*(self: ChatDto): string =
   if self.communityId == "":
     return ""
   return self.id[self.communityId.len .. ^1]
+
+proc updateMissingFields*(chatToUpdate: var ChatDto, oldChat: ChatDto) =
```

Comment:
I moved this here because it is needed by the chat service too, so now both community and chat services can use it.